migration/20180207-2-recreate-materialized-view.sql

```diff
@@ -0,0 +1,133 @@
+drop materialized view mv_works_for_lanes;
+create materialized view mv_works_for_lanes
```

Comment:
When I created `mv_works_for_lanes` I just let `SessionManager.initialize` create it from `materialized_view_for_lanes.sql` on startup. So it should be possible to just drop it and let it get recreated on next startup, rather than drop it and recreate it within the migration script.

This is risky though, because usually there are multiple processes 'starting up' at the same time, and each one might try to create the view.

Dropping and recreating the view within the migration script is also fine, but you'll need to explicitly refresh the view once it's recreated. It's no longer enough for a migration script like this to be a copy of `files/materialized_view_for_lanes.sql`, because `initialize` will only refresh a view that it creates. If the view exists but is empty (as it will be after this script runs), it will stay empty until the next time `refresh_materialized_views` runs.